### PR TITLE
Copy launch description to suite description

### PR DESCRIPTION
**Pull request by kkaarreell**
Fixes https://github.com/teemtee/tmt/issues/2990

Pull Request Checklist

* [x] implement the feature
* [x] include a release note

### CODE REVIEW

File: tmt/steps/report/reportportal.py [hunk 1]
```diff
@@ -380,19 +380,17 @@ def go(self) -> None:
...
             launch_description = self.append_description(launch_description)
 
         # Check whether artifacts URL has been provided
-        if not launch_description:
-            launch_description = self.data.artifacts_url
-        elif self.data.artifacts_url:
-            launch_description = f"{launch_description}, {self.data.artifacts_url}"
+        if self.data.artifacts_url:
+            launch_description += f"<br>{self.data.artifacts_url}"
+            if suite_description:
```
Comment (lukaszachy) [inline tmt/steps/report/reportportal.py:392]: What is the idea behind this 'if'? The line bellow is same as above it - so it just duplicates `<br> artifacts_url`

**Review by lukaszachy — CHANGES_REQUESTED** [review]

File: tmt/steps/report/reportportal.py (hunk 1 above)
Comment (kkaarreell) [reply tmt/steps/report/reportportal.py:392]: OH, wrong variable. Should be setting suite_description instead.

**Comment by kkaarreell** [comment]
JFTR, I have tried various combinations of options `--launch-description`, `--suite-per-plan` and  `--artifacts-url` and in RP it looks as expected.

**Comment by kkaarreell** [comment]
Hi @psss , could you please help me to identify which items from the checklist should be done for this tiny change?

**Review by therazix — APPROVED** [review]

**Comment by thrix** [comment]
@kkaarreell do you have screenshot how it looks in reportportal or provide some reproducer I could try please?

**Review by lukaszachy — APPROVED** [review]

**Comment by kkaarreell** [comment]
> @kkaarreell do you have screenshot how it looks in reportportal or provide some reproducer I could try please?

This is how it looks with 
```
tmt ... --launch-description 'My launch description' --suite-per-plan --artifacts-url 'https://www.foo.cz'
```
Launch:
image
Suite:
image

I could maybe add an extra `<br>` between test plan summary and launch description. WDYT?

**Review by psss — APPROVED** [review]
Looks good, proposing just a minor release note enhancement b3b26fbc.

**Comment by happz** [comment]
Unrelated failures, merging.

**Pull request merged**